pom.xml

```diff
@@ -148,6 +149,11 @@
         <artifactId>commons-codec</artifactId>
         <version>${commons-codec.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.brotli</groupId>
+        <artifactId>dec</artifactId>
```

Comment:
Very bad choice for an artifactId from the brotli team. (not related to this PR)